CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+If you'd like to try or contribute to the plugin, you can clone this repo into your `wp-content/plugins` directory, then activate the plugin. You'll get a separate WordPress menu item called Gutenberg.
+
+Be sure to have <a href="https://nodejs.org/en/">Node installed first</a>. Then, in a terminal, type `npm install` to get the dependencies all set up. Once that finishes, you can type `npm run build` to make the plugin functional, or `npm run dev` to keep the plugin building in the background as you work on it.
```

Comment:
I will make a personal ToDo note to expand this section one of these days. Running out of steam for today :)